fuzz/api/main.go

```diff
@@ -14,9 +15,15 @@ func Fuzz(data []byte) int {
 		return -1
 	}
 
-	// should not panic
-	baker := agent.InitializeTemplateGenerator()
-	baker.GetNodeBootstrappingCmd(&config)
+	baker, err := agent.NewAgentBaker()
+	if err != nil {
+		return -2
```

Comment:
what's the meaning of -2, -3  (and other) return codes? Is this some internal thing for fuzzing lib?

Reply:
should actually just be -1/0/1